acceptance/setup/aio/010_Install.rb

```diff
@@ -29,19 +29,30 @@
 
 step "Install puppetserver..." do
   if ENV['SERVER_VERSION']
-    install_puppetlabs_dev_repo(master, 'puppetserver', ENV['SERVER_VERSION'])
-    install_puppetlabs_dev_repo(master, 'puppet-agent', ENV['SHA'])
-    master.install_package('puppetserver')
+    server_version = ENV['SERVER_VERSION']
+    if master[:hypervisor] == 'ec2' && master[:platform].match(/(?:el|centos|oracle|redhat|scientific)/)
```

Comment:
@MikaelSmith I have attempted to clarify this. Please let me know if it is still confusing.